lib/ngtcp2_conn.c

```diff
@@ -5202,7 +5198,6 @@ static int conn_recv_reset_stream(ngtcp2_conn *conn,
   }
 
   conn->rx.offset += datalen;
-  ngtcp2_conn_extend_max_offset(conn, datalen);
```

Comment:
ngtcp2_strm_rx_offset(strm) may not equal strm->rx.last_offset here, so we can not simply extend connection max offset by strm->rx.last_offset.